rootfs/templates/wrapper-body.sh

```
@@ -249,6 +268,41 @@ function _on_container_exit() {
 	[ -n "${ON_CONTAINER_EXIT}" ] && command -v "${ON_CONTAINER_EXIT}" >/dev/null && "${ON_CONTAINER_EXIT}"
 }
 
+# Call this function to wait for the container to exit, after all other shells have exited.
+function wait_for_container_exit() {
+	local i n shells
+	n=15
+
+	for i in {0..$n}; do
```

Shellcheck (0.10.0)

[warning] Bash doesn't support variables in brace range expansions.

(SC2051)